core/database/PublisherQueryBuilder.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+class PublisherQueryBuilder
+{
+    protected $pdo;
+
+    public function __construct($pdo)
+    {
+        $this->pdo = $pdo;
+    }
+
+    public function selectAllPublishers($table)
```

Comment:
Cia labiau tiktu tokia funkcija naudoti bendrai, pvz padaryti abstract class, pasirasyti joje sita metoda selectAll, kur galetum pasipasuoti table name, ir nesvarbu koks table, visada gauti viska is jos. Tada sitas dvi klases paeextendini su ta, ir naudoji metoda is anos klases. Paidomek inheritance kas yra ir kaip veikia. Tada it would make sense. Bet dabar, kai turi atskira klase tiek vienam tiek kitam, tu jau 100% zinai, kokia cia lentele ir kaip ji vadinasi. Chances are, kad jos pavadinimo tu nekeisi niekada cia, tai hardkodink ir tiek, nes be reikalo apsisunkini sita vieta.